review/communication-react.api.md

```diff
@@ -0,0 +1,1828 @@
+
+// @public (undocumented)
+export const getChatSelector: <Component extends (props: any) => JSX.Element | undefined>(component: Component) => GetChatSelector<Component>;
+
+// @public (undocumented)
+export const GridLayout: (props: GridLayoutProps) => JSX.Element;
+
+// @public (undocumented)
+export interface GridLayoutProps {
+    // (undocumented)
+    children: React_2.ReactNode;
+    // (undocumented)
+    layout?: GridLayoutType;
+    styles?: BaseCustomStylesProps;
+}
+
+// @public (undocumented)
+export type GridLayoutType = 'standard';
+
+// @public (undocumented)
+export const IdentifierContext: React_2.Context<Identifiers>;
+
+// @public (undocumented)
+export const IdentifierProvider: (props: IdentifierProviderProps) => JSX.Element;
+
+// @public (undocumented)
+export interface IdentifierProviderProps {
+    // (undocumented)
+    children: React_2.ReactNode;
+    // (undocumented)
+    identifiers?: Identifiers;
+}
+
+// @public (undocumented)
+export interface Identifiers {
+    messageContent: string;
+    messageTimestamp: string;
+    participantList: string;
+    sendboxTextfield: string;
+    typingIndicator: string;
+    videoGallery: string;
+}
+
+// @public (undocumented)
+export type IncomingCallListener = (event: {
+    callId: string;
+    callerId: string;
+    callerDisplayName?: string;
+    accept: () => Promise<void>;
+    reject: () => Promise<void>;
+}) => Promise<void>;
+
+// @public
+export interface IncomingCallState {
+    callEndReason?: CallEndReason;
+    callerInfo: CallerInfo;
+    endTime: Date | undefined;
+    id: string;
+    startTime: Date;
+}
+
+// @public (undocumented)
+export type IsMuteChangedListener = (event: {
+    identifier: CallIdentifierKinds;
+    isMuted: boolean;
+}) => void;
+
+// @public (undocumented)
+export type IsScreenSharingOnChangedListener = (event: {
+    isScreenSharingOn: boolean;
+}) => void;
+
+// @public (undocumented)
+export type IsSpeakingChangedListener = (event: {
+    identifier: CallIdentifierKinds;
+    isSpeaking: boolean;
+}) => void;
+
+// @public (undocumented)
+export interface JumpToNewMessageButtonProps {
+    // (undocumented)
+    onClick: () => void;
+}
+
+// @public
+export const lightTheme: PartialTheme & CallingTheme;
+
+// @public
+export interface Locale {
+    strings: ComponentStrings;
+}
+
+// @public
+export const LocalizationProvider: (props: LocalizationProviderProps) => JSX.Element;
+
+// @public
+export type LocalizationProviderProps = {
+    locale: Locale;
+    children: React_2.ReactNode;
+};
+
+// @public
+export interface LocalVideoStreamState {
+    mediaStreamType: MediaStreamType;
+    source: VideoDeviceInfo;
+    view?: VideoStreamRendererViewState;
+}
+
+// @public
+export const MeetingComposite: (props: MeetingCompositeProps) => JSX.Element;
```

Comment:
I will move this out of this review (hoping I can tag as @alpha and it will go into its own api file) - the PRs for it won't be ready before wednesday